static/app/views/insights/http/views/httpLandingPage.spec.tsx

```diff
@@ -260,6 +260,8 @@ describe('HTTPLandingPage', function () {
   it('fetches module data', async function () {
     render(<HTTPLandingPage />, {organization});
 
+    await waitForElementToBeRemoved(() => screen.queryAllByTestId('loading-indicator'));
```

Comment:
Oh this was from the PoC where we dynamically loaded the widgets. Still makes sense here that we'd wait for loading components to go away before we check the mocked requests.